app/controllers/admin/reports_controller.rb

```diff
@@ -69,19 +87,25 @@ def reports
           report_title: "Application Details report ",
           report_link: { href: "https://dsdmoj.atlassian.net/wiki/x/JwBOKQE", text: "About this report (opens in new tab)", class: "govuk-link govuk-link--no-visited-state", rel: "noreferrer noopener", target: "_blank" },
           path: :admin_application_details_csv_path,
-          path_text: "Download CSV",
+          path_text: "Download <span class=\"govuk-visually-hidden\">application details</span> CSV".html_safe,
+        },
+        application_digest_download: {
+          report_title: "Application Digest report",
+          report_link: nil,
+          path: :admin_application_digest_csv_path,
+          path_text: "Download <span class=\"govuk-visually-hidden\">application digest</span> CSV".html_safe,
```

Comment:
One too many spaces?!
```suggestion
          path_text: "Download <span class=\"govuk-visually-hidden\">application digest</span>CSV".html_safe,
```